server/server.js

```diff
@@ -24,10 +26,23 @@ if(process.env.NODE_ENV !== 'development') {
     });
 } else {
     const compiler = webpack(config)
-    app.use(webpackMiddleware(compiler));
+    app.use(webpackMiddleware(compiler, {
+        publicPath: config.output.publicPath,
+        stats: {
+            colors: true,
+            assets: false,
+            modules: false,
+        },
+    }));
     app.use(webpackHotMiddleware(compiler));
 }
 
+app.get('*', (req, res) => {
```

Comment:
This catch-all handler should be defined last in order, now requests to endpoints set by `addAuth()` never get executed.